src/set_theory/game/nim.lean

```diff
@@ -182,12 +182,6 @@ begin
   simpa using IH _ (typein_lt_self _)
 end
 
-lemma exists_ordinal_move_left_eq {o : ordinal} (i) : ∃ o' < o, (nim o).move_left i = nim o' :=
-⟨_, typein_lt_self _, move_left_nim' i⟩
-
-lemma exists_move_left_eq {o o' : ordinal} (h : o' < o) : ∃ i, (nim o).move_left i = nim o' :=
-⟨to_left_moves_nim ⟨o', h⟩, by simp⟩
-
```

Comment:
Even though they are no longer used in the proof below, I'm not sure it's a good idea to remove them.

Reply:
These are existence theorems where we can very easily provide the explicit object using `to_left_moves_nim`. So there's no longer a need for them.

Reply:
Okay; I see that [pgame.to_left_moves_nim_symm_lt](https://leanprover-community.github.io/mathlib_docs/set_theory/game/nim.html#pgame.to_left_moves_nim_symm_lt), [pgame.move_left_nim](https://leanprover-community.github.io/mathlib_docs/set_theory/game/nim.html#pgame.move_left_nim) etc. cover these lemmas. 
LGTM
maintainer merge